src/main/java/org/folio/service/accesstypes/AccessTypesServiceImpl.java

```diff
@@ -123,29 +137,41 @@ public CompletableFuture<AccessType> findByCredentialsAndAccessTypeId(String cre
   public CompletableFuture<AccessTypeCollection> findByNames(Collection<String> accessTypeNames,
                                                              String credentialsId,
                                                              Map<String, String> okapiHeaders) {
-    return repository.findByCredentialsAndNames(toUUID(credentialsId), accessTypeNames, tenantId(okapiHeaders))
+    String tenantId = tenantId(okapiHeaders);
+    log.debug("findByNames:: by [accessTypeNames: {}, tenant: {}]", collectionToLogMsg(accessTypeNames), tenantId);
```

Comment:
I guess we could put whole collection in log (it's debug level and it's collection of strings)